.github/workflows/canary.yml

```diff
@@ -7,12 +7,20 @@ on:
 jobs:
   canary-test:
     runs-on: ubuntu-20.04
-    name: Canary Test - ${{ matrix.name }}
+    name: Canary Test - ${{ matrix.aws_region }} - ${{ matrix.name }}
     strategy:
       matrix:
         aws_region:
           # TODO(anuraaga): Add more regions when the layers are published to them.
           - us-east-1
```

Comment:
17 regions in total:
https://github.com/aws-observability/aws-otel-lambda/blob/6c2b69d9c084813ad8716ef4e19302906e4f9ca6/.github/workflows/canary-py.yml#L17

Reply:
Yeah canary won't work until layers are published to them all. Currently this is just using the layer published by soak test, single region in test account